### PR TITLE
[IMP] l10n_mx: insert all account tags from CSV instead of XML

### PR DESCRIPTION
**Pull request by qsb-odoo**
Description of the issue/feature this PR addresses:
    * In order to speedup installation of mx localisation

Current behavior before PR:
    * Account tags are inserted through XML file.

Desired behavior after PR is merged:
    * Account tags will be inserted through CSV file.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_mx/data/account.account.tag.csv [hunk 1]
```diff
@@ -0,0 +1,1083 @@
...
+"account_tag_702_10","702.10 Otros productos financieros","Accounts","","4","Creditable Account"
+"account_tag_703","703 Otros gastos","Accounts","","4","Debitable Account"
+"account_tag_703_01","703.01 Pérdida en venta y/o baja de terrenos","Accounts","","4","Debitable Account"
+"account_tag_703_02","703.02 Pérdida en venta y/o baja de edificios","Accounts","","4","Debitable Account"
+"account_tag_703_03","703.03 Pérdida en venta y/o baja de maquinaria y equipo","Accounts","","4","Debitable Account"
+"account_tag_703_04","703.04 Pérdida en venta y/o baja de automóviles, autobuses, camiones de carga, tractocamiones, montacargas y remolques","Accounts","","4","Debitable Account"
+"account_tag_703_05","703.05 Pérdida en venta y/o baja de mobiliario y equipo de oficina","Accounts","","4","Debitable Account"
+"account_tag_703_06","703.06 Pérdida en venta y/o baja de equipo de cómputo","Accounts","","4","Debitable Account"
+"account_tag_703_07","703.07 Pérdida en venta y/o baja de equipo de comunicación","Accounts","","4","Debitable Account"
+"account_tag_703_08","703.08 Pérdida en venta y/o baja de activos biológicos, vegetales y semovientes","Accounts","","4","Debitable Account"
```
Comment (jco-odoo) [inline addons/l10n_mx/data/account.account.tag.csv:984]: @qsb-odoo Why is this different than in the XML?  "Debitable Accounts" -> "D" and "Accounts" -> "accounts"?  (why the description in stead of the technical field)

**Comment by qsb-odoo** [comment]
@jco-odoo Improved.

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by qsb-odoo** [comment]
@jco-odoo Sorry for the inconveniences that I kept the enterprise PR open. I have closed it now. Can you please r+ again?

**Pull request closed without merging**